doc/040_backup.rst

```diff
@@ -240,6 +240,17 @@ include other filesystems like ``/sys`` and ``/proc``.
 .. note:: ``--one-file-system`` is currently unsupported on Windows, and will
     cause the backup to immediately fail with an error.
 
+Files larger than a given size can be excluded using the `--exclude-larger-than`
+option:
+
+.. code-block:: console
+
+    $ restic -r /srv/restic-repo backup ~/work --exclude-larger-than 1M
+
+This excludes files in ``~/work`` which are larger than 1 MB from the backup.
+The maximum file size should be specified as ``1024``, ``1k``, ``10K``, ``20m``,
+``20M``,  ``30g`` or ``30G``.
```

Comment:
Please add an empty line after `This excludes files in ``~/work`` which are larger than 1 MB from the backup.` and change the following two lines to:

~~~
The default unit for the size value is bytes, so e.g. ``--exclude-larger-than 2048`` would exclude files larger than 2048 bytes (2 kilobytes). To specify other units, suffix the size value with one of ``k``/``K`` for kilobytes, ``m``/``M`` for megabytes, ``g``/``G`` for gigabytes and ``t``/``T`` for terabytes (e.g. ``1k``, ``10K``, ``20m``, ``20M``,  ``30g``, ``30G``, ``2t`` or ``2T``).
~~~